TTHAnalysis/python/plotter/skim_command.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
DONE

Reply:
Yes but the runtime directory depends on $pwd, not on the location of the script.

It can be in differential/ and be run as

sh ttH-multilepton/differential/skim_command.sh

(the same way one runs python ttH-multileptons/ttHplots.py)